scripts/build-win.js

```diff
@@ -0,0 +1,30 @@
+const fs = require('fs');
+const tp = require('teen_process');
+const path = require('path');
+
+const APKS_DIR = './apks';
+const APK_FILENAME = '/ApiDemos-debug.apk';
+
+async function runGradleBuild() {
+  try {
+    await tp.exec('cmd.exe', ['/c', 'gradlew.bat', 'clean', 'assembleDebug']);
+  } catch (e) {
+    throw new Error('Failed at gradle build!\n ' + e);
+  }
+}
+
+async function copyAPK() {
+  try {
+    await fs.promises.access(APKS_DIR, fs.constants.R_OK);
+  } catch (e) {
+    await fs.promises.mkdir(APKS_DIR, { recursive: true });
+  }
+  await fs.promises.copyFile('app/build/outputs/apk/debug/app-debug.apk', path.join('apks', APK_FILENAME));
```

Comment:
`'app/build/outputs/apk/debug/app-debug.apk'` -> `path.resolve('.', 'app', 'build', 'outputs', 'apk', 'debug', 'app-debug.apk')`

Also it makes sence to move this path to constants